docs/content/gs-using-nexus-proc.adoc

```diff
@@ -0,0 +1,35 @@
+[id='gs-using-nexus-proc']
+
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
+
+:nexus-name: Nexus Repository Manager
+
+= Managing Java artifacts using {nexus-name}
+
+The {PRODUCT} cluster includes a running instance of {nexus-name}.
+If you are developing Java applications, you might want to use {nexus-name} to cache and manage dependencies.
+
+== Using {nexus-name} to build your applications
+
+. Add the following to your `application.yaml` file:
```

Comment:
@finp 

This looks right, but looks very specific to when using a Launcher template.
It might be better to make this more generic and say something like:

> Set the following environment variable in your BuildConfig to tell your maven build where Nexus is. `MAVEN_MIRROR_URL=http://nexus.nexus.svc:8081/nexus/content/groups/public`

Reply:
thanks, updated text, but trying to avoid anthropomorphism ;)